package.json

```diff
@@ -97,7 +97,7 @@
     "@emotion/core": "^10.0.35",
     "@svgr/webpack": "^6.2.1",
     "@types/jest": "^28.0.0",
-    "amazon-connect-chatjs": "^2.2.5",
+    "amazon-connect-chatjs": "^3.0.2",
```

Comment:
have we done manual testing for the basic functionalities?

Reply:
yes tested locally, message-receipts are working as expected. 
I did not test interactive-messages and full manual tests.